Makefile.doc

```diff
@@ -218,11 +218,13 @@ doc/refman/html/index.html: doc/refman/Reference-Manual.html $(REFMANPNGFILES) \
 	@touch $(INDEXES)
 	(cd doc/common/styles/html/$(HTMLSTYLE);\
 		for f in `find . -name \*.css`; do \
-			install -m 644 -D $$f ../../../../refman/html/$$f;\
+			$(MKDIR) ../../../../refman/html;\
+			$(INSTALLLIB) $$f ../../../../refman/html;\
```

Comment:
This is the wrong semantics, the $f can be eg `sites/all/themes/coq/coqdoc.css` and then needs to go to `../../../../refman/html/sites/all/themes/coq/coqdoc.css`.
Same for the below hunk (actually I don't know why it's there, it should be redundant since it has a narrower `find`).
Then once this is done the install-doc-html target breaks.